### PR TITLE
Added NZXT phantom 240 case review.

### PR DESCRIPTION
**Pull request by Vogete**
Thought I'd do my part and add a review of my Phantom 240 case 🙂

### CODE REVIEW

File: docs/06-hardware/cases.md [hunk 1]
```diff
@@ -79,3 +79,26 @@ _Lots of space for drives, even with poor cable management._
 * Pricing - I purchased mine on promotion for about $80 years back, but they seem to have nearly doubled in price and sit at around $140 new.
 
 While this chassis is larger than other offerings, I chose it for its generous number of 3.5" bays, included quality fans and support for ATX power supplies. Front I/O is adequate, and the blue power LED can simply be unplugged should it cause a disturbance. Included are two fans that are controlled by a built-in 3 position switch. I've always kept it at its lowest and temperature has never been a concern.
+
+## NZXT Phantom 240
+
+> Review contributed by [Vogete](https://github.com/vogete)
+
+The NZXT Phantom 240 might not stand out from the croud in any of the features of aesthetics it offers (though subjectically it looks amazing in white with its stormtrooper style), but it will definitely impress you with its price. At just $69.99 (release price), it's hard to beat the value of the materials you get with it. Unfortuantely as far as I can tell it's not being sold much at most places, so it might be hard, or impossible to come by.
```
Comment (ironicbadger) [inline docs/06-hardware/cases.md:87]: Spell check "unfortunately".

File: docs/06-hardware/cases.md [hunk 2]
```diff
@@ -79,3 +79,26 @@ _Lots of space for drives, even with poor cable management._
...
+![nzxt-phantom-240](../images/hardware/cases/phantom-240.jpg)
+_Plenty of space and airflow, but the dust can find its way in as well._
+
+* Motherboard Form Factor - Mini-ITX, mATX, ATX
+* PSU Form Factor - ATX
+* Number of 3.5" drive bays - 6 (0 hotswap)
+* Dimensions - 7.6 x 20.8 x 20.8" / 195 x 530 x 529 mm
+* Pricing - It was released for $69.99 (I purchased it for $50 in 2018), but haven't found any recent prices as nobody really sells them anymore.
+
+The case comes with 3.5" drive bays, sadly none of them are hot swap, and another 3 5.25" bays which you can technically repurpose for even more drives. The 3.5" bays are removable, so if you prefer to fit a longer graphics card, you have the option to reevalute your priorities.
```
Comment (ironicbadger) [inline docs/06-hardware/cases.md:98]: The case comes with ?how many? 3.5" bays

**Review by ironicbadger — CHANGES_REQUESTED** [review]

File: docs/06-hardware/cases.md (hunk 1 above)
Comment (Vogete) [reply docs/06-hardware/cases.md]: I spell checked everything now and fixed all my mistakes

File: docs/06-hardware/cases.md (hunk 2 above)
Comment (Vogete) [reply docs/06-hardware/cases.md]: apparently I forgot to write the number. added it now.

**Review by ironicbadger — APPROVED** [review]

**Pull request merged**